Integrations/ThreatConnect/ThreatConnect.py

```diff
@@ -1299,6 +1299,195 @@ def tc_get_groups():
     )
 
 
+def get_group_request(group_type, group_id):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/{}/{}'.format(group_type, group_id))
+
+    return tc.api_request(ro).json()
+
+
+def get_group():
+    """
+    retrieve a single Group
+    """
+    group_type = demisto.args().get('group_type')
+    group_id = int(demisto.args().get('group_id'))
+
+    response = get_group_request(group_type, group_id)
+    if group_type == 'adversaries':
+        data = response.get('data', {}).get('adversarie', {})
+    if group_type == 'campaigns':
+        data = response.get('data', {}).get('campaign', {})
+    if group_type == 'documents':
+        data = response.get('data', {}).get('document', {})
+    if group_type == 'emails':
+        data = response.get('data', {}).get('email', {})
+    if group_type == 'events':
+        data = response.get('data', {}).get('event', {})
+    if group_type == 'incidents':
+        data = response.get('data', {}).get('incident', {})
+    if group_type == 'intrusionSets':
+        data = response.get('data', {}).get('intrusionSet', {})
+    if group_type == 'reports':
+        data = response.get('data', {}).get('report', {})
+    if group_type == 'signatures':
+        data = response.get('data', {}).get('signature', {})
+    if group_type == 'threats':
+        data = response.get('data', {}).get('threat', {})
+
+    if response.get('status') == 'Success':
+        contents = {
+            'ID': data.get('id'),
+            'Name': data.get('name'),
+            'Owner': data.get('owner'),
+            'DateAdded': data.get('dateAdded'),
+            'EventDate': data.get('eventDate'),
+            'Status': data.get('status')
```

Comment:
why do we need to return the status ?

Reply:
It's not the 'Success' status, it's the status for the group like: "Needs Review"